File: src/main/java/com/devspace/conexfy/converters/ConStringToConHttpMethodEnumReadingConverter.java
package com.devspace.conexfy.converters;

import org.springframework.core.convert.converter.Converter;
import org.springframework.data.convert.ReadingConverter;
import org.springframework.lang.NonNull;

import com.devspace.conexfy.enums.ConHttpMethodEnum;



@ReadingConverter
public class ConStringToConHttpMethodEnumReadingConverter implements Converter<String, ConHttpMethodEnum> {
    @Override
    public ConHttpMethodEnum convert(@NonNull String source) {
        return ConHttpMethodEnum.valueOf(source);
    }
}

File: src/main/kotlin/com/devspace/conexfy/converters/ConStringToConHttpMethodEnumReadingConverter.kt
package com.devspace.conexfy.converters

import com.devspace.conexfy.enums.ConHttpMethodEnum
import org.springframework.core.convert.converter.Converter
import org.springframework.data.convert.ReadingConverter
import org.springframework.lang.NonNull

@ReadingConverter
class ConStringToConHttpMethodEnumReadingConverter : Converter<String, ConHttpMethodEnum> {
    override fun convert(@NonNull source: String): ConHttpMethodEnum {
        return ConHttpMethodEnum.valueOf(source)
    }
}